s3_file_field/__init__.py

```diff
@@ -4,3 +4,9 @@
 from .fields import S3FileField  # noqa: F401
 
 default_app_config = 's3_file_field.apps.S3FileFieldConfig'
+
+version = {}
+with open("s3_file_field/version.py") as fp:
+    exec(fp.read(), version)
+
+__version__ = version["__version__"]
```

Comment:
Why this approach and not?:

```py
from .version import __version__
```

----

Also, using `exec` in the package is frowned upon